### DISCUSSION TITLE
直播间玩法又要更新了(粉丝团-小心心功能升级公告)

### DISCUSSION THREAD

**Discussion by Janet-Baker** [andywang425/BLTH#250] [General]
[https://link.bilibili.com/p/eden/news#/newsdetail?id=2780](https://link.bilibili.com/p/eden/news#/newsdetail?id=2780)

# 粉丝团-小心心功能升级公告

2022-05-13 14:48:23

各位B站的UP主和用户，你们好。
 
为了给大家提供更有趣的直播互动体验，我们将在5月16日开始对粉丝团的部分功能做调整。你将看到一些全新的、有趣的直播互动玩法。调整详情如下，希望你能喜欢。
 
## 调整详情

1. 小心心将在5月16日-5月22日期间暂停获取，在5月16日之前用户已获得的小心心不受影响可正常使用，请在23日之前尽快使用。
2.  小心心将在5月23日起调整为直播间专属道具，即在A直播间获取的小心心只能在A直播间生效，同时不再作为礼物道具送出，主播将自动获得用户在本直播间内通过任务获取的小心心（用户与主播同时获得）。
3. 粉丝团任务将于5月23日更新，会陆续提供更多与主播互动的玩法，并有更多手段获取小心心与主播亲密度。

附小心心更新之后获取方式
 
在已加入主播粉丝团的直播间中，当主播开播时，完成以下任务就能获得对应的小心心。
 

- 发送5条弹幕：+20小心心/弹幕

- 发送5个表情包：+20小心心/表情包

- 连续观看30分钟（累计）：+100小心心/5分钟

- 给主播点赞三次：+100小心心 

- 送100电池的礼物：+1小心心/1电池

- 分享5次直播间：+150小心心

新版小心心的有效期为自然周内生效，每周日24点清零。

目前仅支持移动端，仅能在勋章对应的直播间获得小心心，即在A主播直播间里加入粉丝团，需要在A直播间通过完成任务获得小心心，且仅能在A直播间生效。小心心能在直播间的互动玩法中消耗。

**Comment by andywang425** [comment]
玩法更新后脚本也会更新的。不过快期末了，可能要等到暑假才能更新脚本。
